coordinator/changefeed/changefeed.go

```diff
@@ -171,6 +173,10 @@ func (c *Changefeed) IsMQSink() bool {
 	return c.isMQSink
 }
 
+func (c *Changefeed) IsStorageSink() bool {
```

Comment:
It's better to use a enum to help decide whether it's MQSink, StorageSink or mysqlSink.